docs/device-api.md

```diff
@@ -125,6 +125,12 @@ specification:
 |                        | track_dirty_pages |    O     |       O        |      O       |        O         |     O      |      O       |
 |                        | vcpu_count        |    O     |       O        |      O       |        O         |     O      |      O       |
 
+## Known device limitations
+
+If more than 64 devices are configured for a VM in total on aarch64, only first
+64 of them are functional
+([related issue](https://github.com/firecracker-microvm/firecracker/issues/4207)).
+
```

Comment:
Shouldn't this be consolidated in `README.md::## Known issues and Limitations`, with the other limitations?

Reply:
we intentionally got rid of that section, because it was hard to maintain. We took an approach where we document limitations where they naturally belong (see https://github.com/firecracker-microvm/firecracker/blob/main/docs/jailer.md#known-limitations , https://github.com/firecracker-microvm/firecracker/blob/main/docs/hugepages.md#known-limitations )

Reply:
Ah ok, thx for the explanation!